.drone.yml

```diff
@@ -124,6 +124,27 @@ steps:
     target:
     - nightly
 
+- name: nightly_documentation
+  pull: default
+  image: lampepfl/dotty:2020-01-22-2
+  depends_on:
+  - test
+  - test_bootstrapped
+  - community_build
+  - test_java11
+  - publish_nightly
+  commands:
+  - ./project/scripts/genDocs -doc-snapshot
```

Comment:
We should not have a new snapshot of each nightly
```
https://dotty.epfl.ch/0.22.0-bin-20200114-193f7de-NIGHTLY/...
https://dotty.epfl.ch/0.22.0-bin-20200115-aaaaaaa-NIGHTLY/...
https://dotty.epfl.ch/0.22.0-bin-20200116-bbbbbbb-NIGHTLY/...
...
```

Instead, we should replace the latest nightly documentation on each nightly release
```
https://dotty.epfl.ch/nightly/...
```

Reply:
It looks like the simplest way is to do it in `project/scripts/genDocs` after doing `mv 0.*/ $PWD/docs/_site` we can rename `$PWD/docs/_site/0.22.0-bin-20200115-aaaaaaa-NIGHTLY` to `$PWD/docs/_site/nightly`